src/elements/content-sidebar/withSidebarAnnotations.js

```diff
@@ -130,6 +134,31 @@ export default function withSidebarAnnotations(
             }
         }
 
+        updateAnnotation() {
+            const {
+                annotatorState: { action, annotation },
+                api,
+                file,
+                isOpen,
+                location,
+            } = this.props;
+
+            const feedAPI = api.getFeedAPI(false);
```

Comment:
This method duplicates quite a bit of logic from `addAnnotation`. Can we simplify/reuse the common parts?

Reply:
I've extracted the logic for refreshing Activity Sidebar into a `refreshActivitySidebar` method.